execution/executor-benchmark/src/block_partitioning.rs

```diff
@@ -17,10 +17,16 @@ pub(crate) struct BlockPartitioningStage {
 }
 
 impl BlockPartitioningStage {
-    pub fn new(num_shards: usize, partitioner_config: &dyn PartitionerConfig) -> Self {
-        let maybe_partitioner = if num_shards <= 1 {
+    pub fn new(
+        num_shards: usize,
+        partitioner_config: &dyn PartitionerConfig,
+        force_partitioning: bool,
```

Comment:
Hmm not sure why we need this, can we just use the partitioner when sharding is enabled and it is enabled when we set the number of shards (even if it is set to 1 ?) and set the default value to 0 (which means off)

Reply:
Yeah, we can do this by setting default to 0 and use that for unsharded. Just wanted to keep the delta in the cmd usage minimal. But I guess nobody specifies num_shards for unsharded execution, so change should be ok.